tox.ini

```diff
@@ -15,6 +15,10 @@ envdir = {toxworkdir}/py36
 commands=
     ; Force packaging even if setup.{py,cfg} haven't changed
     ; rm -rf ./*.egg-info/
+    
+    ; temporary until we can install a new version > 2021.1 (depends on the release)
```

Comment:
@italovalcy @ajoaoff with this temporary workaround that we'll revert on issue #7 once we have a new `python-openflow` release the tests are deterministically passing now. execution with `tox` for instance. @italovalcy let me know if you could also please re-run the tests that failed in the container: 

```
❯ tox
coverage create: /mnt/b/repos/noviflow/.tox/py36
coverage installdeps: -Urrequirements/dev.txt
coverage installed: astroid==2.9.0,backcall==0.1.0,backports.entry-points-selectable==1.1.1,black==21.11b1,click==8.0.3,coverage==6.2,dataclasses==0.8,decorator==4.4.2,distlib==0.3.3,doc
opt==0.6.2,docutils==0.18.1,filelock==3.4.0,Flask==1.1.2,Flask-Cors==3.0.8,Flask-SocketIO==4.2.1,importlib-metadata==4.8.2,importlib-resources==5.4.0,ipython==7.13.0,ipython-genutils==0.
2.0,isort==5.10.1,itsdangerous==1.1.0,janus==0.4.0,jedi==0.16.0,Jinja2==2.11.1,-e git+https://github.com/kytos-ng/kytos.git@f8912ef9e3baa66062373442550b18d737ca14b9#egg=kytos,-e git+ssh:
//git@github.com/kytos-ng/noviflow.git@b32178c079380578c33c7cc859289f8950a07819#egg=kytos_noviflow,-e git+https://github.com/kytos-ng/of_core.git@b4fc04c8620b0db5136778a17d7bea7d33e05f2c
#egg=kytos_of_core,lazy-object-proxy==1.6.0,lockfile==0.12.2,MarkupSafe==1.1.1,mccabe==0.6.1,mypy-extensions==0.4.3,packaging==21.3,parso==0.6.2,pathspec==0.9.0,pathtools==0.1.2,pep517==
0.12.0,pexpect==4.8.0,pickleshare==0.7.5,pip-tools==6.4.0,platformdirs==2.4.0,pluggy==1.0.0,prompt-toolkit==3.0.5,ptyprocess==0.6.0,py==1.11.0,pycodestyle==2.8.0,Pygments==2.10.0,PyJWT==
1.7.1,pylint==2.12.1,pyparsing==3.0.6,python-daemon==2.2.4,python-engineio==3.12.1,-e git+ssh://git@github.com/kytos-ng/noviflow.git@b32178c079380578c33c7cc859289f8950a07819#egg=python_o
penflow,python-socketio==4.5.1,regex==2021.11.10,six==1.16.0,toml==0.10.2,tomli==1.2.2,tox==3.24.4,traitlets==4.3.3,typed-ast==1.5.0,typing_extensions==4.0.1,virtualenv==20.10.0,watchdog
==0.10.2,wcwidth==0.1.9,Werkzeug==1.0.1,wrapt==1.13.3,yala==3.1.0,zipp==3.6.0
coverage run-test-pre: PYTHONHASHSEED='3448533101'
coverage run-test: commands[0] | pip install git+https://github.com/kytos-ng/python-openflow.git
Collecting git+https://github.com/kytos-ng/python-openflow.git
  Cloning https://github.com/kytos-ng/python-openflow.git to /tmp/pip-req-build-ohf8nqfz
  Running command git clone --filter=blob:none -q https://github.com/kytos-ng/python-openflow.git /tmp/pip-req-build-ohf8nqfz
  Resolved https://github.com/kytos-ng/python-openflow.git to commit abd40982e03e09bddabdbf19f0aa7dee9b616f05
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: python-openflow
  Building wheel for python-openflow (setup.py) ... done
  Created wheel for python-openflow: filename=python_openflow-2021.1-py3-none-any.whl size=198885 sha256=7d4574380b33faee26d38067255740431f92f8172d9ed1b2cd8f701c7198a873
  Stored in directory: /tmp/pip-ephem-wheel-cache-ajbd9ju2/wheels/a8/e7/65/8015039c0f3b19fcf62d6800f3c79240907d02c4183dc921da
Successfully built python-openflow
Installing collected packages: python-openflow
  Attempting uninstall: python-openflow
    Found existing installation: python-openflow 2021.1
    Uninstalling python-openflow-2021.1:
      Successfully uninstalled python-openflow-2021.1
Successfully installed python-openflow-2021.1
coverage run-test: commands[1] | python setup.py coverage
/mnt/b/repos/noviflow/.tox/py36/lib/python3.6/site-packages/setuptools/__init__.py:151: SetuptoolsDeprecationWarning: setup_requires is deprecated. Supply build dependencies using PEP 51
7 pyproject.toml build-requires.
  SetuptoolsDeprecationWarning,
running coverage
/mnt/b/repos/noviflow/.tox/py36/lib/python3.6/site-packages/setuptools/__init__.py:151: SetuptoolsDeprecationWarning: setup_requires is deprecated. Supply build dependencies using PEP 51
7 pyproject.toml build-requires.
  SetuptoolsDeprecationWarning,
running pytest
/mnt/b/repos/noviflow/.tox/py36/lib/python3.6/site-packages/setuptools/command/easy_install.py:159: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip a
nd other standards-based tools.
  EasyInstallDeprecationWarning,
/mnt/b/repos/noviflow/.tox/py36/lib/python3.6/site-packages/setuptools/command/install.py:37: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other st
andards-based tools.
  setuptools.SetuptoolsDeprecationWarning,
Searching for pytest
Best match: pytest 6.2.5
Processing pytest-6.2.5-py3.6.egg

Using /mnt/b/repos/noviflow/.eggs/pytest-6.2.5-py3.6.egg
Searching for iniconfig
Best match: iniconfig 1.1.1
Processing iniconfig-1.1.1-py3.6.egg

Using /mnt/b/repos/noviflow/.eggs/iniconfig-1.1.1-py3.6.egg
Searching for attrs>=19.2.0
Best match: attrs 21.2.0
Processing attrs-21.2.0-py3.6.egg

Using /mnt/b/repos/noviflow/.eggs/attrs-21.2.0-py3.6.egg
running egg_info
writing kytos_noviflow.egg-info/PKG-INFO
writing dependency_links to kytos_noviflow.egg-info/dependency_links.txt
writing requirements to kytos_noviflow.egg-info/requires.txt
writing top-level names to kytos_noviflow.egg-info/top_level.txt
reading manifest file 'kytos_noviflow.egg-info/SOURCES.txt'
adding license file 'LICENSE'
writing manifest file 'kytos_noviflow.egg-info/SOURCES.txt'
running build_ext
================================================================================== test session starts ===================================================================================
platform linux -- Python 3.6.6, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
cachedir: .tox/py36/.pytest_cache
rootdir: /mnt/b/repos/noviflow, configfile: setup.cfg
collected 5 items

tests/unit/test_main.py .....                                                                                                                                                      [100%]

==================================================================================== warnings summary ====================================================================================
tests/unit/test_main.py::TestMain::test__eq__success_with_different_flows
tests/unit/test_main.py::TestMain::test__eq__success_with_equal_flows
tests/unit/test_main.py::TestMain::test_create_noviactions
tests/unit/test_main.py::TestMain::test_noviaction_experimenter_pack_unpack
tests/unit/test_main.py::TestMain::test_noviaction_set_bfd_data
  /mnt/b/repos/noviflow/.tox/py36/src/kytos/kytos/core/config.py:167: UserWarning: Unknown arguments: ['tests/unit']
    warnings.warn(f"Unknown arguments: {unknown}")

-- Docs: https://docs.pytest.org/en/stable/warnings.html
============================================================================= 5 passed, 5 warnings in 3.68s ==============================================================================
Name                      Stmts   Miss  Cover
---------------------------------------------
__init__.py                   0      0   100%
main.py                      14      2    86%
of_core/v0x04/action.py      42     10    76%
pyof/v0x04/action.py         64      1    98%
settings.py                   0      0   100%
---------------------------------------------
TOTAL                       120     13    89%
lint installed: astroid==2.9.0,backcall==0.1.0,backports.entry-points-selectable==1.1.1,black==21.11b1,click==8.0.3,coverage==6.2,dataclasses==0.8,decorator==4.4.2,distlib==0.3.3,docopt=
=0.6.2,docutils==0.18.1,filelock==3.4.0,Flask==1.1.2,Flask-Cors==3.0.8,Flask-SocketIO==4.2.1,importlib-metadata==4.8.2,importlib-resources==5.4.0,ipython==7.13.0,ipython-genutils==0.2.0,
isort==5.10.1,itsdangerous==1.1.0,janus==0.4.0,jedi==0.16.0,Jinja2==2.11.1,-e git+https://github.com/kytos-ng/kytos.git@f8912ef9e3baa66062373442550b18d737ca14b9#egg=kytos,-e git+ssh://gi
t@github.com/kytos-ng/noviflow.git@b32178c079380578c33c7cc859289f8950a07819#egg=kytos_noviflow,-e git+https://github.com/kytos-ng/of_core.git@b4fc04c8620b0db5136778a17d7bea7d33e05f2c#egg
=kytos_of_core,lazy-object-proxy==1.6.0,lockfile==0.12.2,MarkupSafe==1.1.1,mccabe==0.6.1,mypy-extensions==0.4.3,packaging==21.3,parso==0.6.2,pathspec==0.9.0,pathtools==0.1.2,pep517==0.12
.0,pexpect==4.8.0,pickleshare==0.7.5,pip-tools==6.4.0,platformdirs==2.4.0,pluggy==1.0.0,prompt-toolkit==3.0.5,ptyprocess==0.6.0,py==1.11.0,pycodestyle==2.8.0,Pygments==2.10.0,PyJWT==1.7.
1,pylint==2.12.1,pyparsing==3.0.6,python-daemon==2.2.4,python-engineio==3.12.1,python-openflow @ git+https://github.com/kytos-ng/python-openflow.git@abd40982e03e09bddabdbf19f0aa7dee9b616
f05,python-socketio==4.5.1,regex==2021.11.10,six==1.16.0,toml==0.10.2,tomli==1.2.2,tox==3.24.4,traitlets==4.3.3,typed-ast==1.5.0,typing_extensions==4.0.1,virtualenv==20.10.0,watchdog==0.
10.2,wcwidth==0.1.9,Werkzeug==1.0.1,wrapt==1.13.3,yala==3.1.0,zipp==3.6.0
lint run-test-pre: PYTHONHASHSEED='3448533101'
lint run-test: commands[0] | python setup.py lint
/mnt/b/repos/noviflow/.tox/py36/lib/python3.6/site-packages/setuptools/__init__.py:151: SetuptoolsDeprecationWarning: setup_requires is deprecated. Supply build dependencies using PEP 51
7 pyproject.toml build-requires.
  SetuptoolsDeprecationWarning,
running lint
Yala is running. It may take several seconds...
WARNING: pylint # TODO enable black later to minimize changes is not a valid linter
INFO: Finished pycodestyle
INFO: Finished isort
:) No issues found.
________________________________________________________________________________________ summary _________________________________________________________________________________________
  coverage: commands succeeded
  lint: commands succeeded
  congratulations :)
```